azure-advanced/Packer/dbscripts/mysql-analytics.sql

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
@rasanjaya85 as per official [documentation](https://apim.docs.wso2.com/en/latest/learn/analytics/configuring-apim-analytics/#step-4-configure-databases), we may not need this DB table. Shall we remove it?

Reply:
I think this cannot be deleted since it's been used by the Dashboard deployment.toml file. so +1 for keeping this.

Reply:
@rasanjaya85 yes you must keep the database. But, we no longer need to create a table. AFAIK, this is automatically done at the startup.

Reply:
Ok. Shall we keep this as it is?  If the product doesn't create the table then Terraform does create the table after the DB instance is provisioned.

Reply:
Yes, let us keep it for now.